src/main/java/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator.java

```diff
@@ -60,6 +60,10 @@ public void setTheme(ThemeManagerFactory theme) {
     }
 
     public ThemeManagerFactory getTheme() {
+        if (theme == null) {
+            return new NoOpThemeManagerFactory();
```

Comment:
Is there a way of detecting if Dark theme is installed and defaulting to its system variant? (That way admins wouldn't need to configure it)

Reply:
I think dark theme would have to do that. I think it would configure itself if the default if none is set. This change might break that though as it can't tell between none and 'Light'